Src/TestFramework/Shared/StubMetricProcessor.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.ApplicationInsights.TestFramework
+{
+    using System;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+    public class StubMetricProcessor : IMetricProcessor
+    {
+        public Action<Metric, double> OnTrack = (metric, value) => { };
```

Comment:
nitpick: even test code should have comments for consistency and readability of future maintainers.